gfop/get_food_counts.py

```diff
@@ -15,25 +15,28 @@ def load_food_metadata():
     return gfop_metadata
 
 
-def get_sample_types(simple_complex='all'):
+def get_sample_types(simple_complex='all', level=0):
     """
     Return:
         Global FoodOmics ontology containing only simple, only complex, or all foods.
     Args:
         simple_complex (string): one of 'simple', 'complex', or 'all'.
                                  Simple foods are single ingredients while complex foods contain multiple ingredients.
                                  'all' will return both simple and complex foods.
+        level (integer): indicates the level of the food ontology to use.
+                         One of 1, 2, 3, 4, 5, 6, or 0.
+                         0 will return counts for individual reference spectrum files, rather than food categories.
     """
     gfop_metadata = load_food_metadata()
     if simple_complex != 'all':
         gfop_metadata = gfop_metadata[gfop_metadata['simple_complex'] == simple_complex]
-    col_sample_types = ['sample_name'] + [f'sample_type_group{i}' for i in range(1, 7)]
-    return (gfop_metadata[['filename', *col_sample_types]]
-            .set_index('filename'))
+    # Select food hierarchy level.
+    col_sample_types = [f'sample_type_group{level}' if level > 0 else 'sample_name']
```

Comment:
Now this is only a single column, so you don't need to put it in a list. Just use the column label directly. Also remove the `*` on the next line then.